java-checks/src/main/resources/org/sonar/l10n/java/rules/java/S6541.html

```diff
@@ -1,18 +1,18 @@
-<p>Methods should not perform too many tasks (Brain method).</p>
+<p>Methods should not perform too many tasks (Brain Method).</p>
 <h2>Why is this an issue?</h2>
-<p>This issue is often referred as 'Brain method' anti-pattern. <a href="https://link.springer.com/book/10.1007/3-540-39538-5">Defined here</a> <br> A
-brain method is a method that tends to centralize its owner class logic and generally performs too many operations. This can include checking too many
-conditions, using lots of variables, and ultimately making it difficult to understand, maintain and reuse.<br> It is characterized by high LOC number,
-high cyclomatic and cognitive complexity, and a large number of variables being used.</p>
+<p>This issue is raised when Sonar considers that a method is a 'Brain Method'. <br> A Brain Method is a method that tends to centralize its owner
```

Comment:
I would rewrite this sentence just for clarity: This issue is raised when Sonar considers a method a 'Brain Method'.

Reply:
I'll keep this as a note in the rspec PR!